src/material/datepicker/multi-year-view.ts

```diff
@@ -218,6 +218,21 @@ export class MatMultiYearView<D> implements AfterContentInit, OnDestroy {
     );
   }
 
+  _yearBecomesActive(event: MatCalendarUserEvent<number>) {
```

Comment:
This is nearly identical to the `_yearSelected` method above. I'm open to suggestions on how to refactor this, or if we should refactor it to share code.